src/main/resources/templates/subscription.html

```diff
@@ -251,32 +251,32 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                     </select>
                                 </div>
                             </div>
-                            <div class="p-1 border form-group" data-bind="visible: authenticationType() == 'BASIC_AUTH'">
+                            <div class="p-1 border form-group" data-bind="visible: authenticationType() == 'BASIC_AUTH' || authenticationType() == 'BASIC_AUTH_JENKINS' ">
                                 <label class="pt-1 control-label font-weight-bold">Username</label>
                                 <div>
                                     <input id="userNameInput" title="Enter user name" data-bind="textInput:userName" name="userName"
                                         placeholder="userName" class="form-control" type="text" />
                                     <span class="help-block"></span>
                                 </div>
 
-                                <label data-bind="visible: $root.showPassword()" class="pl-1 control-label font-weight-bold">Password*</label>
+                                <label data-bind="visible: $root.showPassword(), text: authenticationType() == 'BASIC_AUTH' ? 'Password*' : 'API-token'" class="pl-1 control-label font-weight-bold"></label>
                                 <div>
                                     <input id="passwordInput" title="Enter password" data-bind="visible: $root.showPassword() && ($root.mode() == 'edit' || $root.mode() == 'add'),textInput:$data.password"
                                         name="password" placeholder="password" class="form-control" type="password" />
```

Comment:
Should this not change content to token when it change from Password* to API-token?